packaging/rules

```diff
@@ -6,8 +6,8 @@ RPMBUILD=rpmbuild --define "_topdir `pwd`/packaging" --define "cccver $(VERSION)
 help:
 	@echo "Targets:"
 	@echo "    tgz:   Build a cellcc-v*.tar.gz source tarball"
-	@echo "    srpm7: Build an srpm for EL7"
-	@echo "    rpm7:  Build an rpm for EL7"
+	@echo "    srpm: Build an srpm for EL or Fedora"
+	@echo "    rpm:  Build an rpm for EL or Fedora"
```

Comment:
I do want to mention in here what platforms the packaging "supports" (so someone isn't surprised if this doesn't work on rhel6 or suse or whatever). but this may be better as a separate note after all of these, like "rpm packaging currently supported for RHEL/CentOS 7-8, and Fedora 33-34"